ecl/eclagent/eclagent.ipp

```diff
@@ -512,6 +512,7 @@ public:
     virtual IUserDescriptor *queryUserDescriptor();
     virtual void selectCluster(const char * cluster);
     virtual void restoreCluster();
+    virtual unsigned getElapsedMs() const override {UNIMPLEMENTED;} //Implementing for roxie first, not sure what this means from an eclagent perspective.  ECL plugin implies elapsed time of entire query
```

Comment:
I've created https://track.hpccsystems.com/browse/HPCC-29490 to consider implementing for thor/hthor.